src/pages/tBTC/Bridge/TbtcBalanceCard.tsx

```diff
@@ -1,22 +1,15 @@
 import { FC, ComponentProps } from "react"
 import TokenBalanceCard from "../../../components/TokenBalanceCard"
 import { Token } from "../../../enums"
-import { Card, Box } from "@threshold-network/components"
+import { Card } from "@threshold-network/components"
 
 export const TbtcBalanceCard: FC<ComponentProps<typeof Card>> = ({
   ...restProps
 }) => {
   return (
     <TokenBalanceCard
       token={Token.TBTC}
-      title={
-        <>
-          <Box as="span" textTransform="lowercase">
-            t
-          </Box>
-          btc balance
-        </>
-      }
+      title="tBTC Balance"
       tokenSymbol={"tBTC"}
       withSymbol={true}
       withHigherPrecision={true}
```

Comment:
Now I see something problematic about removing the `withHigherPrecision` flag. The way it worked before was that the `higherPrecision` is an optional value, and if it's not provided, the default value of it would be 2 (it is set in `InlineTokenBalance`). Now, you could pass `withHigherPrecision` flag to component and it would still display the tooltip (that is placed in `InlineTokenBalance`) even if you don't pass the `precision` prop. I agree that it is counter-intuitive though.

Because of the thing that I'v described above, the higher precision tooltip is not displayed for tbtc balance that is in the top layout section.

(1st solution) We can fix that by passing `higherPrecision={2}` to `TokenBalanceCard` in `TbtcBalanceCard`. By removing the `withHigherPrecision` property we will have to always pass a `higherPrecision` property to display the tooltip, which too might be little counter-intuitive, as we would have to think about what number should be passed there instead just passing a true value and let the component handle it itself, but I think this solution would be good enough.

```suggestion
      higherPrecision={2}
```

(2nd solution) We could also create additional property, which would be `withHigherPrecisionTooltip` and if it's set to true we could display the tooltip in `InlineTokenBalance`. So, in `TbtcBalanceCard` component:

```
{shouldRenderTokenAmount ? (
            (withHigherPrecisionTooltip || precision) ? (
              <InlineTokenBalance
                tokenAmount={tokenAmount}
                tokenFormat={tokenFormat}
                tokenDecimals={tokenDecimals}
                precision={precision}
                withHigherPrecision
              />
            ) : (
              _tokenAmount
            )
          ) : (
            "--"
          )}
```

I think this is why we've had `withHigherPrecision` property in the first place. We would not need to pass the `higherPrecision` property then.

I'll let you decide which fix you prefer. Both are ok.

<img width="188" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/a3ad9080-5346-425d-bb74-fd95659770ca">

Reply:
Ref commit: 87358e2d

Reply:
Conversation continued here: https://github.com/threshold-network/bitcoin-on-base/pull/17#discussion_r1489804397